docs/azure.md

```diff
@@ -0,0 +1,178 @@
+## Setup the environment
+
+The current demo is done on MicroK8s, but this can also be done on other substrates, like AKS. 
+
+### Setup AKS
+
+Install the `azure-cli` tool
+
+```shell
+sudo snap install azure-cli --channel latest/stable
```

Comment:
```suggestion
sudo snap install azcli --channel latest/stable
```

`azure-cli` is a different snap, does not seem maintained

Reply:
ooooopsss, great catch! really! not sure how I messed this up, since in my local system I indeed only have `azcli` installed